diesel_codegen/src/schema_inference/sqlite.rs

```diff
@@ -45,7 +45,7 @@ pub fn determine_column_type(cx: &mut ExtCtxt, attr: &ColumnInformation) -> P<as
     } else if is_double(&type_name) {
         quote_ty!(cx, ::diesel::types::Double)
     } else {
-        cx.span_err(cx.original_span(), &format!("Unsupported type: {}", type_name));
+        cx.span_err(cx.expansion_cause(), &format!("Unsupported type: {}", type_name));
```

Comment:
I'm not sure this is the right change. It's might be `cx.call_site()` or even something else (I'm not familiar with rust syntax extensions). This was broken by https://github.com/rust-lang/rust/commit/4259fba7e6b87966099a2df04e75b678e26e0811, FYI.